pkg/cvo/sync_worker.go

```diff
@@ -402,6 +402,12 @@ func (w *SyncWorker) syncPayload(ctx context.Context, work *SyncWork,
 		if w.payload != nil {
 			implicitlyEnabledCaps = payload.GetImplicitlyEnabledCapabilities(payloadUpdate.Manifests, w.payload.Manifests,
 				work.Capabilities)
+			if strings.HasPrefix(payloadUpdate.Release.Version, "4.14.") {
+				deploymentConfig := configv1.ClusterVersionCapability("DeploymentConfig")
+				if _, ok := work.Capabilities.EnabledCapabilities[deploymentConfig]; !ok && !capability.Contains(implicitlyEnabledCaps, deploymentConfig) {
+					implicitlyEnabledCaps = append(implicitlyEnabledCaps, deploymentConfig)
```

Comment:
@wking The issue is we have declared a capability but there are no associated manifests which has this capability. We can have this issue in future as well. Can we declare a generic code structure for this? A method which can return capability names which should be enabled when no manifests has the capability. I would expect the code structure to be present in master and then we should back-port this appropriately.

Reply:
@bparees ack. Will add the labels

Reply:
I've created [OTA-1040](https://issues.redhat.com/browse/OTA-1040) trying to feel out a long-term plan for this kind of thing.